spring-auto-restdocs-core/src/main/java/capital/scalable/restdocs/snippet/StandardTableSnippet.java

```diff
@@ -140,7 +140,7 @@ private String joinAndFormat(String description, List<String> constraints,
 
         res.append(constr.toString());
 
-        return res.toString();
+        return res.toString().replace("|", "\\|");
```

Comment:
General questions is on which characters to escape, e.g. * _ &. We never had an issue so far and thus I think that it's fine to not add them for now.

Reply:
this one is important only in tables. Of course more generic escaping would be beneficial